routing/src/user/userService.ts

```diff
@@ -0,0 +1,43 @@
+import fs from 'fs';
+
+export function addUser(data: string): number {
+  if (data === '') {
+    return 400;
```

Comment:
Service should not return any status codes) It's controller's work.
See example of project structure https://github.com/EQuimper/nodejs-api-boilerplate/tree/master/src